tests/utils/test_showtools.py

```
@@ -290,3 +292,32 @@ def test_show_task_core_tools_ordered():
+    assert "siliconcompiler.utils.showtools" not in sys.modules, "showtools imported early"
+    print(ShowTask.get_task("vcd").tool())
+"""
+
+
+@pytest.mark.parametrize("available,expected", [
+    ("surfer", "surfer"),
+    ("gtkwave", "gtkwave")
+])
+def test_vcd_viewer_preference(available, expected):
+    """The installed VCD viewer is preferred, with no plugins involved."""
+    proc = subprocess.run([sys.executable, "-c", VCD_PREFERENCE_PROBE, available],
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)